docs/content/usage/windows-machine-images.md

```diff
@@ -0,0 +1,383 @@
+## Overview
+
+Wakame has functionality
```

Comment:
In official documentation like this, we should always refer to Wakame as Wakame-vdc. This of course goes for the rest of the guide too.